contract/r/gnoswap/gov/governance/proposal_test.gno

```diff
@@ -284,11 +283,11 @@ func TestUpdateProposalsState(t *testing.T) {
 			proposals = avl.NewTree()
 
 			proposal := tt.setupProposal(tt.currentTime)
-			proposals.Set(fomatUint(1), proposal)
+			proposals.Set(formatUint(uint32(1)), proposal)
```

Comment:
I think we don't need to covert type here
```suggestion
			proposals.Set(formatUint(1), proposal)
```

Reply:
It needs to be, if you run test file you can see it panics without converting